src/pages/components/navigation/code.js

```diff
@@ -32,6 +32,9 @@ class NavigationCode extends React.Component {
             <div className="example-container relative">
 
               <header className="header mb-space-m">
+                <button type="button" class="mobile-menu_toggle" style={{ position: "absolute" }}>
```

Comment:
We can probably move this "position: 'absolute'" to the actual class, otherwise approved! Good work!